retail-managment-system/rms-ui/src/app/pages/product/product.component.html

```diff
@@ -86,32 +85,34 @@ <h3 class="panel-title fw-bold">
             </td>
           </ng-container>
 
-          <tr mat-header-row *matHeaderRowDef="displayedColumns"></tr>
+          <tr mat-header-row *matHeaderRowDef="displayedColumns;sticky:true;"></tr>
```

Comment:
Why do you need that "sticky:true" ?

Reply:
To make the header row in table sticky while scrolling
![chrome-capture-2023-1-15](https://user-images.githubusercontent.com/116714827/219000650-ac485489-3507-41ab-a3fb-5911419cb921.gif)

Reply:
ok